codemagic.yaml

```
@@ -0,0 +1,485 @@
+definitions: 
+  instance_mac_mini_m2: &instance_mac_mini_m2
+    instance_type: mac_mini_m2
+    max_build_duration: 30
+
+  env_versions: &env_versions
+    flutter: 3.22.3
+    xcode: 15.4 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)